test/solc/CommandLineParser.cpp

```diff
@@ -81,6 +81,7 @@ BOOST_AUTO_TEST_CASE(no_options)
 	expectedOptions.modelChecker.initialize = true;
 	expectedOptions.modelChecker.settings = {
 		ModelCheckerContracts::Default(),
+		/*divModWithSlacks*/true,
 		ModelCheckerEngine::None(),
 		false,
 		smtutil::SMTSolverChoice::All(),
```

Comment:
Maybe `expectedOptions.modelChecker.settings = {}` would work here as you suggested in one of my PRs.

Reply:
Done